pkg/codegen/pcl/local.go

```diff
@@ -36,6 +36,9 @@ func (lv *LocalVariable) SyntaxNode() hclsyntax.Node {
 }
 
 func (lv *LocalVariable) Traverse(traverser hcl.Traverser) (model.Traversable, hcl.Diagnostics) {
+	if lv == nil || lv.Type() == nil {
```

Comment:
Can lv itself be nil here?

Reply:
I'm not sure tbh, though I don't think so 🤔